retreat.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: event-page
+title: Retreat
+event_date: 2025
+location: Studio Friction, Denver
+registration_status: open
+featured_image: assets/images/studio-friction.jpg
```

Comment:
RE: image handling

This should change to `image: studio-friction.jpg` and we can place the file in `img/`